<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCreateAcc</class>
 <widget class="QWidget" name="FormCreateAcc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Account:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="srcAccountID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>32</y>
     <width>641</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="srcAccountPrivate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="genNewSign">
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>90</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>641</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign of signer:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
